app/views/devise/registrations/new.html.erb

```diff
@@ -19,11 +19,15 @@
         <%= f.email_field :email, class: "center-form form-control", placeholder: "Email"  %>
         </div>
 
+        <div class="styled-select">
+        <%= f.select :organization, [['College','college'],['MedSchool','med'],['EdSchool','ed']], placeholder: "Select Organization" %>
```

Comment:
This is a static list of `organization`s. It would have to be updated in code every time the administrator (or a user) of AskUp wanted to add, remove, or edit an organization. Let's hook it up to the backend and show the available organizations, instead of a hard-coded list.